openpype/hosts/maya/plugins/load/load_image.py

```diff
@@ -0,0 +1,176 @@
+    place3d = cmds.shadingNode("place3dTexture", asUtility=True,
+                               name="place3d")
+
+    cmds.connectAttr(place3d + '.worldInverseMatrix[0]',
+                     projection + ".placementMatrix")
+    cmds.connectAttr(file + '.outColor', projection + ".image")
+
+    return file, place, projection, place3d
+
+
+def create_stencil():
+    """Create texture with extra place2dTexture offset and stencil
+
+    Mimics Maya "file [Stencil]" creation.
+    """
+
+    file, place = create_texture()
+
+    place_stencil = cmds.shadingNode("place2dTexture", asUtility=True,
+                                     name="place2d_stencil")
+    stencil = cmds.shadingNode("stencil", asTexture=True, name="stencil")
+
+    for src_attr, dest_attr in [
+        ("outUV", "uvCoord"),
+        ("outUvFilterSize", "uvFilterSize")
+    ]:
+        src_plug = "{}.{}".format(place_stencil, src_attr)
+        cmds.connectAttr(src_plug, "{}.{}".format(place, dest_attr))
+        cmds.connectAttr(src_plug, "{}.{}".format(stencil, dest_attr))
+
+    return file, place, stencil, place_stencil
+
+
+class FileNodeLoader(load.LoaderPlugin):
+    """File node loader."""
+    # TODO: Implement color space manamagent OCIO (set correct color space)
+
+    families = ["image", "plate", "render"]
+    label = "Load file node"
+    representations = ["exr", "tif", "png", "jpg"]
+    icon = "image"
+    color = "orange"
+    order = 2
+
+    options = [
+        EnumDef(
+            "mode",
+            items={
+                "texture": "Texture",
+                "projection": "Projection",
+                "stencil": "Stencil"
+            },
+            default="texture",
+            label="Texture Mode"
+        )
+    ]
+
+    def load(self, context, name, namespace, data):
+
+        path = self.fname
+        asset = context['asset']['name']
+        namespace = namespace or unique_namespace(
+            asset + "_",
+            prefix="_" if asset[0].isdigit() else "",
+            suffix="_",
+        )
+
+        with namespaced(namespace, new=True) as namespace:
+            # Create the nodes within the namespace
+            nodes = {
+                "texture": create_texture,
+                "projection": create_projection,
+                "stencil": create_stencil
+            }[data.get("mode", "texture")]()
+
+            # Set the file node attributes
+            file_node = cmds.ls(nodes, type="file")[0]
+            cmds.setAttr(file_node + ".fileTextureName", path, type="string")
+
+            # Set UV tiling mode if UDIM tiles
+            # TODO: Detect UDIM tiles and set accordingly (also on update)
+            cmds.setAttr(file_node + ".uvTilingMode", 3)    # UDIM-tiles
+
+            # Enable sequence if publish has `startFrame` and `endFrame` and
+            # `startFrame != endFrame`
+            # TODO: Detect sequences (also on update)
+            # cmds.setAttr(file_node + ".useFrameExtension", True)
```

Comment:
Meanwhile, i am thinking about something like this 
```
version = context["version"]
version_data = version.get("data", {})
frameStart = version_data.get("frameStart")
frameEnd = version_data.get("frameStart")
if frameStart != frameEnd:
      cmds.setAttr(file_node + ".useFrameExtension", 1)
else:
      cmds.setAttr(file_node + ".useFrameExtension", 0)
```
but also I am thinking about if udim and framerange will mix up at the same time.

Reply:
I think we should somehow be able to do something like:

```python
is_frame = representation.get('frame') is not None
is_udim = representation.get('udim') is not None
```

I recall **udim** and **frame** getting stored with the representations - but not sure where it's stored. @iLLiCiTiT might know what _the_ way is to detect whether a publish is frame dependent and/or udim based or not.

Reply:
As far as i know, the frame comes before the udim in terms of the file naming convention. 
Something like this : {published_file_name}.{frame}_{udim}.{ext}. 
e.g.: workfile_v001.1230_1001.exr

Reply:
We store first frame and first udim (if were used). `repre_doc["context"].get("frame")` or `repre_doc["context"].get("udim")`

Reply:
This should be implemented now - the code is a bit much maybe. Let me know what you think. It's trying to be as explicit as possible about udim and frames if they are defined in the original template upon publishing.

This logic would NOT work for any file that is published with a `{originalFilename}` template instead. @iLLiCiTiT @mkolar thoughts?